content/docs/contributing/release-process.md

```diff
@@ -417,23 +417,10 @@ page if a step is missing or if it is outdated.
      cd ../..
 
      find . -name go.mod -not -path ./_bin/\* -exec dirname '{}' \; | xargs -L1 -I@ sh -c 'cd @; go mod tidy'
-     git add **/go.mod **/go.sum
-     git commit -m"Update cmd/ctl's go.mod to $RELEASE_VERSION"
+     git add "**/go.mod" "**/go.sum"
```

Comment:
Without the quotes, the files didn't get added.

```sh
$ # Must be run from the cert-manager repo folder.
cd cmd/ctl
go get github.com/cert-manager/cert-manager@$RELEASE_VERSION
cd ../..

find . -name go.mod -not -path ./_bin/\* -exec dirname '{}' \; | xargs -L1 -I@ sh -c 'cd @; go mod tidy'
git add **/go.mod **/go.sum
git commit --signoff -m"Update cmd/ctl's go.mod to $RELEASE_VERSION"
xargs: warning: options --max-lines and --replace/-I/-i are mutually exclusive, ignoring previous --max-lines value
On branch update-cmd/ctl/v1.12.7
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   cmd/ctl/go.mod
        modified:   cmd/ctl/go.sum
        modified:   test/integration/go.mod

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        .envrc
        cert-manager/
        release-notes.md

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
My fault: I forgot that ** is specific to zsh. I didn’t realize that git add also supported ** 😅